src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/DataContractJsonSerializer.cs

```diff
@@ -75,11 +74,6 @@ public DataContractJsonSerializer(Type type, DataContractJsonSerializerSettings
             _serializer = new DataContractJsonSerializerImpl(type, settings);
         }
 
-        public bool IgnoreExtensionDataObject
-        {
-            get { return _ignoreExtensionDataObject; }
-        }
```

Comment:
This is public surface area.  It is not dead code.  Please revert.

Reply:
@MattKotsenas did you use the linker (or data from linker - #17905) to find this? It should not flag public surface area.

Reply:
I was starting with the linker data from that issue, but was also using source tools to pull out dead locals. That tool turned out to be overzealous and was removing public members if they had no usage. I went back and scrubbed before commiting, but missed this one (along with the `CodeObject` contstrutor @stephentoub caught). Sorry for the extra noise!